pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1382,10 +1382,6 @@ def test_format_timedelta_ticks_narrow(self):
         for l, l_expected in zip(labels, expected_labels):
             assert l.get_text() == l_expected
 
-    @td.xfail_if_mpl_2_2
```

Comment:
we originally had the mac skips on here (e.g. before 2.2). not really sure if they are still relevant though.

Reply:
I'm testing on a mac, so hopefully we're OK.